packages/terra-clinical-result/src/common/other/OtherTemplates.module.scss

```diff
@@ -0,0 +1,24 @@
+
+
+:local {
+  .entered-in-error {
+    color: var(--terra-clinical-result-entered-in-error-color);
+    display: inline;
+    font-size: var(--terra-clinical-result-entered-in-error-font-size, 1em); // Must use ems for font rescaling
```

Comment:
Are other themes changing these font sizes or can we break this out to reduce the number of theme variables that need maintained?